features/feed.feature

```diff
@@ -36,18 +36,18 @@ Feature: Feed
     And the feed does not contain "Article1"
 
   Scenario: Feed only includes posts
-    Given a "Create(Note)" Activity "Note1" by "Alice"
-    And "Alice" sends "Note1" to the Inbox
-    And "Note1" is in our Inbox
+    Given a "Create(Note)" Activity "CreateNote1(Note1)" by "Alice"
```

Comment:
Im not grokking the verbose naming of the Activity?

Reply:
So the problem here is that we weren't actually creating a `Like(Note)` we were creating a `Like(Create(Note))` - which allowed the tests to pass, but wasn't actually correct behaviour - so the verbose naming here is to allow us to refer to `Note1` as the object inside the activity `CreateNote1`